engine/audio/src/audiocapture.cpp

```diff
@@ -227,12 +223,20 @@ void AudioCapture::processData()
 #endif
 
     // 5 ********* Calculate the average signal power
-    m_signalPower = pwrSum / m_captureSize;
+    foreach(int barsNumber, m_fftMagnitudeMap.keys())
+    {
+        double bandMagnitude = fillBandsData(barsNumber);
+        pwrSum += bandMagnitude;
+        if (maxMagnitude < bandMagnitude)
+            maxMagnitude = bandMagnitude;
+    }
```

Comment:
This block computes the total power in all bands (pwrSum) and also picks out the band with maximum magnitude.